README.adoc

```diff
@@ -0,0 +1,61 @@
+= IoT OpenShift Demo
+
+:Author:    Andrew Block, Ishu Verma
+:Email:     ablock@redhat.com, iverma@redhat.com
+:Date:      10/10/2016
+
+== Overview
+Demonstration of Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Gateway enables IoT data transformation, routing and real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Each component of the demo runs in its own container, thereby, allowing security, scalability and ease of deployment.
+
+image:images/ose-iot.png[]
```

Comment:
Do we want to denote these are not full instances of BRMS and Fuse and instead xPaaS?